hledger-web/hledger-web.m4.md

```diff
@@ -119,6 +123,12 @@ You can use `--host` to change this, eg `--host 0.0.0.0` to listen on all config
 Similarly, use `--port` to set a TCP port other than 5000, eg if you are
 running multiple hledger-web instances.
 
+Both of these options are ignored when `--socket` is used. In this case, it creates
+an `AF_UNIX` socket file at the supplied path and uses that for communication.
+This can be used to run multiple hledger-web instances on the same host, eg
+behind a reverse proxy.
+This option also implies `--serve`.
+
```

Comment:
I think just the first sentence is enough here; the explanation of --socket should be at that flag.